charts/chart.line.js

```diff
@@ -128,6 +128,10 @@ line.prototype.getSpec = function() {
 
 
 function getLineMark(config, metadata){
+    interpolation = "linear";
```

Comment:
@chamilad  Add default value to util file and add same behaviour for area chart as well